tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -37,6 +37,8 @@
     no_log: true
     with_items:
       - "OVESETUP_CONFIG/adminPassword=str:{{ he_admin_password }}"
+      - "OVESETUP_DWH_DB/password=str:{{ ovirt_engine_setup_dwh_db_password | default('123456') }}"
+      - "OVESETUP_DB/password=str:{{ ovirt_engine_setup_db_password  | default('123456') }}"
```

Comment:
In this way all the new deployment will be configured with DB passwords == 123456 if a custom password is not explicitly set and this is definitively a bad idea. 
Adding OVESETUP_DWH_DB/password and OVESETUP_DB/password only when we have custom values is definitively a better idea.

Reply:
Thanks @tiraboschi 

How do you like this: 

```
- name: Add lines to answerfile
    lineinfile:
      path: /root/ovirt-engine-answers
      line: "{{ item }}"
    no_log: true
    with_items:
      - "OVESETUP_CONFIG/adminPassword=str:{{ he_admin_password }}"
  - name: Add DB and DWH Password to answerfile when defined
    lineinfile:
      path: /root/ovirt-engine-answers
      line: "{{ item }}"
    no_log: true
    with_items:
      - "OVESETUP_DWH_DB/password=str:{{ ovirt_engine_setup_dwh_db_password }}"
      - "OVESETUP_DB/password=str:{{ ovirt_engine_setup_db_password  }}"
    when:
      ovirt_engine_setup_dwh_db_password is defined and ovirt_engine_setup_db_password is defined

```

If you like this approach I will update my PR.

Reply:
This is going to work only if ovirt_engine_setup_dwh_db_password and ovirt_engine_setup_db_password are defined at the same time.
I think you can simply pass OVESETUP_DWH_DB/password=none:None when ovirt_engine_setup_dwh_db_password is not defined

Reply:
I'm also wondering if we can even do something smarter with all the ovirt_engine_setup_ prefixed values...

Reply:
@tiraboschi I tried leaving `default()` and `default(none:None)` and also `default('none:None')` 
All these did give incorrect results/failure. 
I am looking at more alternatives.

Reply:
@tiraboschi @irosenzw Updated my PR please take a look at the way I think we can handle this.